lib/src/main/java/growthbook/sdk/java/GBFeaturesRepository.java

```diff
@@ -44,7 +44,7 @@ public class GBFeaturesRepository implements IGBFeaturesRepository {
 
     @Nullable
     @Getter
-    private final String encryptionKey;
+    private final String decryptionKey;
```

Comment:
I agree with this change, but it will break any existing implementation that is using the old naming.  I think we should have both `decryptionKey` and `encryptionKey` getters/setters work and mark the old encryptionKey ones as deprecated.  Then, in a future version we can remove them.

Reply:
Good catch, it would be a breaking change. I'll fix it.